truenas/webui
# NAS-125655 / 24.04 / Submit additional fields when submitting a feedback

**Testing**

1. Click Feedback indicator in the top bar
2. Fill in Feedback Dialog and submit

**Expected:** New fields `product_type` and `product_model` are submitted to  `/api/reviews/add/`
